### PR TITLE
UCM: Fix mmap events test

### PR DESCRIPTION
**Pull request by yosefe**
Fixes segfault with valgrind on Jazz machine

### CODE REVIEW

File: src/ucm/mmap/install.c [hunk 1]
```diff
@@ -86,8 +87,7 @@ void ucm_fire_mmap_events(int events)
     }
 
     if (events & (UCM_EVENT_SBRK|UCM_EVENT_VM_MAPPED|UCM_EVENT_VM_UNMAPPED)) {
-        (void)sbrk(ucm_get_page_size());
-        (void)sbrk(-ucm_get_page_size());
+        (void)sbrk(0);
```
Comment (hoopoepg) [inline src/ucm/mmap/install.c]: sbrk(0) will not produce vm_map/vm_unmap events

Comment (yosefe) [reply src/ucm/mmap/install.c]: i'm relying on other tests to provide it.. 
however i could probably update the 'if' and add comment to document this

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/10445/ for details (Mellanox internal link).

**Comment by yosefe** [comment]
blocked by #3647

**Comment by yosefe** [comment]
@hoopoepg can you pls take a look?

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/10555/ for details (Mellanox internal link).

**Comment by yosefe** [comment]
bot:retest

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/10619/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/10625/ for details (Mellanox internal link).

**Pull request merged**